element-hq/element-x-android
# Implement MSC4027 (Custom images in reactions)

Fixes https://github.com/element-hq/element-x-android/issues/2159
 
## Type of change


## Content

Implement rendering of custom images in reaction per MSC4027

## Motivation and context

Ran into mxc reactions while testing Element X and wanted to make it work

## Screenshots / GIFs

![image](https://github.com/element-hq/element-x-android/assets/683652/9afa70d3-e82f-4e7f-8597-c887f10cafb5)



## Tests



- Send custom reaction using Cinny
- Observe the reaction showing up in Element X

## Tested devices

- OS version(s):
Android 6, 13, 14

## Checklist
